prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -82,7 +130,7 @@ class LivdTableDownload : CliktCommand(
         // Get the link to the LIVD-SARS-CoV-2-yyyy-MM-dd.xlsx file
         val livdFile = search(cdcLOINCTestCodeMappingPageUrl, livdSARSCov2File)
         if (livdFile.isEmpty()) {
-            TermUi.echo("\tError: unable to find LOINC code data to download!")
+            TermUi.echo("\tERROR: unable to find LOINC code data to download!")
```

Comment:
Comment on writing to a file: At this point, we do not care about keeping a file since the data is uploaded to the database, so perhaps just use a temp file or something like that.

Reply:
Done.